### DISCUSSION TITLE
Fire from a deeplink

### DISCUSSION THREAD

**Discussion by kurtisnelson** [ueman/feedback#99] [Q&A]
What is the best way to fire the feedback form from a deeplink? I want to be able to get just text from the user. Nesting the  `Betterfeedback.of().show()` directly in a widget violates the build patterns. Is there a different widget to use to get just the text form, but as configured?

**Comment by ueman** [comment]
> What is the best way to fire the feedback form from a deeplink?

I'm not quite sure I understand what you're trying to say. 

>  I want to be able to get just text from the user. Nesting the `Betterfeedback.of().show()` directly in a widget violates the build patterns. 

You can use a Builder to do that. You just use a Builder as a child of BetterFeedback. Use the builders context to show the feedback view.

> Is there a different widget to use to get just the text form, but as configured?

This package currently does not allow to get feedback without an image.
